src/http.rs

```diff
@@ -83,8 +95,17 @@ where
         &self,
         endpoint: &str,
         content: T,
+        interval: Option<u64>,
```

Comment:
similarly each post request can decide on the interval and even opt out of it (e.g. registration call is not invoked on interval)